example.html

```diff
@@ -4,6 +4,28 @@
 ---
 
 Clicking on the example will open a static version of the example, powered by <a href="https://nbviewer.jupyter.org/">nbviewer</a>.
+To interact with a "live" version, it is possible from there to download the notebook and run it locally:
```

Comment:
BTW how about converting this file from HTML to markdown (one can embed HTML into a `md` file, so the `<div class="gallery_example">` stuff at the end could probably stay unchanged).

At least for me, editing Markdown feels more convenient. But that's taste, so don't feel obliged to do this.

Reply:
Yes absolutely! I feel quite uncomfortable writing HTML when Markdown would seem to suffice. I will investigate how to do that (but not in this PR).